packages/@aws-cdk/aws-stepfunctions/README.md

```diff
@@ -508,6 +508,23 @@ new stepfunctions.StateMachine(stack, 'MyStateMachine', {
 });
 ```
 
+## Import
+
+Any Step Functions state machine that has been created outside the stack can be imported
+into your CDK stack.
+
+State machines can be imported by their ARN via the `StateMachine.fromStateMachineArn()` API
+
+```ts
+import * as sfn from 'aws-stepfunctions';
+
+const stack = new Stack(app, 'MyStack');
+const stateMachine = sfn.StateMachine.fromStateMachineArn(
```

Comment:
```suggestion
sfn.StateMachine.fromStateMachineArn(
```

Reply:
merged this suggestion.

However, I'd argue that the typical usage when importing a resource is to assign it to a variable so that it can be used subsequently in code. It's not quite the same as defining a new resource which will be synthesized and created on a deploy. Do you see things differently?